### ISSUE TITLE
Bad packaging in the `pkcs11-v1.3` release.

### ISSUE THREAD

**Issue by xyakimo1** [GoogleCloudPlatform/kms-integrations#24]
There seems to be an issue in the tar archives in the `PKCS #11 Library v1.3` release. To be more precise, paths of the files begin with `../../Downloads/`. Steps to reproduce:

```
cd /tmp
wget https://github.com/GoogleCloudPlatform/kms-integrations/releases/download/pkcs11-v1.3/libkmsp11-1.3-linux-amd64.tar.gz
tar -xvf libkmsp11-1.3-linux-amd64.tar.gz
```

Output:

```
tar: Removing leading `../' from member names
tar: ../Downloads/._libkmsp11-1.3-linux-amd64: Member name contains '..'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.macl'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/: Member name contains '..'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/kmsp11.h: Member name contains '..'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/._.DS_Store: Member name contains '..'
tar: Ignoring unknown extended header keyword 'SCHILY.fflags'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.FinderInfo'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/.DS_Store: Member name contains '..'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/._LICENSE: Member name contains '..'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.quarantine'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.metadata:kMDItemWhereFroms'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.macl'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/LICENSE: Member name contains '..'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/._libkmsp11.so.sig: Member name contains '..'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.quarantine'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.metadata:kMDItemWhereFroms'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/libkmsp11.so.sig: Member name contains '..'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/._NOTICE: Member name contains '..'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.quarantine'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.metadata:kMDItemWhereFroms'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.macl'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.lastuseddate#PS'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/NOTICE: Member name contains '..'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/._libkmsp11.so: Member name contains '..'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.quarantine'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.metadata:kMDItemWhereFroms'
tar: ../Downloads/libkmsp11-1.3-linux-amd64/libkmsp11.so: Member name contains '..'
tar: Exiting with failure status due to previous errors
```

There is no such issue in the v1.2 release.

**Comment by bdhess** [comment]
Thanks for the report.  We've pulled the release, and we'll re-release when this issue is fixed.

**Comment by tdbhacks** [comment]
We have just re-released v1.3, the issue should now be fixed.
